crates/sui-core/src/authority/epoch_start_configuration.rs

```diff
@@ -107,7 +115,12 @@ pub struct EpochStartConfigurationV4 {
     /// Do the state objects exist at the beginning of the epoch?
     authenticator_obj_initial_shared_version: Option<SequenceNumber>,
     randomness_obj_initial_shared_version: Option<SequenceNumber>,
-    // TODO: In a new version, make these initial shared versions a vector.
+}
+
+#[derive(Serialize, Deserialize, Debug, Eq, PartialEq)]
+pub struct EpochStartConfigurationV5 {
+    v4: EpochStartConfigurationV4,
```

Comment:
I guess you are trying to save effort here but I feel like this is more confusing than following the existing pattern

Reply:
Fair point. Addressed both feedback